extensions/measurement-tracking/src/viewports/ViewportOverlay.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cornerstone from 'cornerstone-core';
+import classnames from 'classnames';
+
+const ViewportOverlay = ({
+  imageId,
+  scale,
+  windowWidth,
+  windowCenter,
+  imageIndex,
+  stackSize,
+  activeTools
+}) => {
+  const topLeft = 'top-viewport left-viewport';
+  const topRight = 'top-viewport right-viewport-scrollbar';
+  const bottomRight = 'bottom-viewport right-viewport-scrollbar';
+  const bottomLeft = 'bottom-viewport left-viewport';
```

Comment:
I'm not sure I understand what any of these classes are used for. Are there some baked in classes somewhere that fall outside of tailwind?

Reply:
These classes are used to apply the correct spacing between viewport and the text + font styles. if we use any fallback from rcw it will be different then whats in zeplin